tests/foreman/cli/test_errata.py

```diff
@@ -565,12 +1430,36 @@ def test_positive_list_filter_by_org_label(self):
         1. erratum list --organization-label=<org_label>
 
         @Assert: Errata is filtered by Org label.
-
-        @caseautomation: notautomated
-
         """
-
-    @stubbed()
+        org_erratum_list = Erratum.list({
+            'organization-label': self.org['label'],
+            'per-page': 1000
+        })
+        org_errata_ids = {
+            errata['errata-id']
+            for errata in org_erratum_list
+        }
+        errata_org_erratum_list = Erratum.list({
+            'organization-label': self.errata_org['label'],
+            'per-page': 1000
+        })
+        errata_org_errata_ids = {
+            errata['errata-id']
+            for errata in errata_org_erratum_list
+        }
+        self.assertEqual(len(org_errata_ids), self.org_product_erratum_count)
```

Comment:
thanks @renzon for your comments, the primary purpose of this PR is to transfer the tests from 6.3 to 6.2 and to add host collection tests that cannot be tested in 6.3 because of a bug, this module has been refactored already, but for the moment any refactoring should be done in a separate PR  in 6.3 and after than cherry-picked back to 6.2.z